win32/src/win32apimodule.cpp

```diff
@@ -5390,6 +5390,50 @@ static PyObject * PyLoadString(PyObject *self, PyObject *args)
 	return rc;
 }
 
+// @pymethod PyHANDLE|win32api|LoadResource|Finds a resource in a PE file. Returns a HRSRC(HANDLE) to the resource if found, otherwise returns None.
+static PyObject * PyFindResource(PyObject *self, PyObject *args) {
+
+	PyObject *ret;
+
+	PyObject *obhModule;
+	PyObject *obName;
+	PyObject *obType;
+
+	HMODULE hModule;
+	LPTSTR lpName = NULL;
+	LPTSTR lpType = NULL;
+
+	if ( !PyArg_ParseTuple(args, "OOO:FindResource",
+			&obhModule, // @pyparm <o PyHANDLE>|handle||The handle of the module containing the resource.  Use None for currrent process executable.
+			&obName,    // @pyparm <o PyResourceId>|name||Name of the resource to find
+			&obType     // @pyparm <o PyResourceId>|type||The type of resource to find.
+		))
+		return NULL;
+
+
+	if (PyWinObject_AsHANDLE(obhModule, (HANDLE *)&hModule)
+		&& PyWinObject_AsResourceId(obType, &lpType) 
+		&& PyWinObject_AsResourceId(obName, &lpName)) {
+			HRSRC resource = FindResource(hModule, lpName, lpType);
+
+			if (resource) {
+				ret = PyWinObject_FromHANDLE(resource);
+			} else {
+				Py_INCREF(Py_None);
+				ret = Py_None;
+			}
+
+
+		} else {
```

Comment:
the indentation here is still very strange - this `else` line should be at the same indentation as the `if`